tests/unittests/aggregation_tests.py

```
@@ -1615,6 +1616,521 @@ def test_expr(self):
+    def test_select_prevents_partitioning(self):
+        sample = self._dataset.first()
+        view = self._dataset.select(sample.id)
+        self.assertEqual(self._get_num_partitions(view), 1)
+
+
+class PartitionedAggregationCorrectnessTests(unittest.TestCase):
+    """Verifies that partitioned (8 partitions) aggregation results match
+    non-partitioned (1 partition) results exactly, including order."""
+
+    @classmethod
+    def setUpClass(cls):
```

Ruff (0.15.9)

[warning] Missing return type annotation for classmethod `setUpClass`

Add return type annotation: `None`

(ANN206)